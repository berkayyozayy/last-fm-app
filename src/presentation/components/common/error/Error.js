import React from "react";
import classes from "./Error.module.css";

function Error({ message }) {
  return (
    <div className={classes.error}>
      <h3>{message}</h3>
    </div>
  );
}

export default Error;
