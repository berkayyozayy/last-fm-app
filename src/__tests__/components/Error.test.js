import { render } from "@testing-library/react";
import Error from "../../presentation/components/common/error/Error";

describe("error component", () => {
  test("it should render", () => {
    const { container } = render(<Error />);
    expect(container).toMatchSnapshot();
  });
});
