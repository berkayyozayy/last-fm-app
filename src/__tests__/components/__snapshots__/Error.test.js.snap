// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error component it should render 1`] = `
<div>
  <div
    class="error"
  >
    <h3 />
  </div>
</div>
`;
